packages/noya-renderer/src/components/EditablePath.tsx

```diff
@@ -51,6 +51,47 @@ function EditablePathPoint({ point, fill, stroke }: EditablePathPointProps) {
   );
 }
 
+interface EditableControlPointProps {
+  point: Point;
+  controlPoint: Point;
+  fill: Paint;
+  stroke: Paint;
+}
+
+function ControlPathPoint({
+  point,
+  controlPoint,
+  fill,
+  stroke,
+}: EditableControlPointProps) {
+  const { CanvasKit } = useReactCanvasKit();
+
+  const path = useMemo(() => {
+    const path = new CanvasKit.Path();
+
+    path.addRect(
+      CanvasKit.XYWHRect(
+        controlPoint.x - POINT_SIZE / 2 + 2,
+        controlPoint.y - POINT_SIZE / 2 + 2,
+        POINT_SIZE / 2,
+        POINT_SIZE / 2,
+      ),
+    );
+
+    return path;
+  }, [CanvasKit, controlPoint.x, controlPoint.y]);
+  useDeletable(path);
+  const points = [point, controlPoint];
+
+  return (
+    <>
+      <Polyline points={points} paint={stroke} />
+      <Path path={path} paint={fill} />
```

Comment:
We should use `<Rect>` instead of `<Path>` here, both because it's simpler, and so that we can avoid instantiating a `CanvasKit.Path`